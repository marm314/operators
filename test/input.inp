   +1 bai baj bck bcl
