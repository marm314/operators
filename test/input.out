   +1 bck bcl bai baj 	
   -1 bck baj 	* delta_il
   +1 bck bai 	* delta_jl
   +1 bcl baj 	* delta_ik
   -1 	* delta_ikdelta_jl
   -1 bcl bai 	* delta_jk
   +1 	* delta_ildelta_jk
